components/observatorium.libsonnet

```diff
@@ -224,6 +224,19 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
       namespace: obs.config.namespace,
       replicas: 1,
       commonLabels+:: obs.config.commonLabels,
+      metrics: {
+        readEndpoint: 'http://%s.%s.svc.cluster.local:%d' % [
+          obs.queryCache.service.metadata.name,
+          obs.queryCache.service.metadata.namespace,
+          obs.queryCache.service.spec.ports[0].port,
+        ],
+        writeEndpoint: 'http://%s.%s.svc.cluster.local:%d' % [
+          obs.receiveService.metadata.name,
+          obs.receiveService.metadata.namespace,
+          obs.receiveService.spec.ports[2].port,
+        ],
+      },
+    } + if std.length(obs.config.loki) != 0 then {
```

Comment:
nit. prefer the more visual function `std.objectHas(o, f)`

> Returns true if the given object has the field (given as a string), otherwise false. Raises an error if the arguments are not object and string respectively. Returns false if the field is hidden.

Reply:
Thanks @periklis.  For this case, I think there is an empty `loki` object. because
```
  loki+:: if std.objectHas(cr.spec, 'loki') then {
    image: if std.objectHas(cr.spec.loki, 'image') then cr.spec.loki.image else default.loki.image,
    replicas: if std.objectHas(cr.spec.loki, 'replicas') then cr.spec.loki.replicas else default.loki.replicas,
    version: if std.objectHas(cr.spec.loki, 'version') then cr.spec.loki.version else default.loki.version,
    objectStorageConfig: if lokiObjectStorageConfig != null then lokiObjectStorageConfig else default.loki.objectStorageConfig,
  } else {},
```
correct me If I am wrong. Thanks.

Reply:
let's move forward with this PR, it's just a nit nothing special.